.github/workflows/ui.yml

```diff
@@ -0,0 +1,25 @@
+name: UI
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "packages/ui/**"
+  pull_request:
+    paths:
+      - "packages/ui/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
```

Comment:
This CICD file does nothing? It just checks out the repo, there is no tests or anything, I don't think it's needed

Reply:
There is nothing yet to test. I just got it ready... but I can remove it if you want

Reply:
It will add one minute to our CICD times with no benefit right now, but I don't think that's the end of the world.

Reply:
i'd say we can remove if we don't need it now, or comment it out.